app/controllers/application_controller.rb

```diff
@@ -7,10 +7,11 @@ class ApplicationController < ActionController::Base
   protected
 
   def devise_configure_permitted_parameters
-    devise_parameter_sanitizer.permit :invite, keys: [:email, :role, :registers]
+    devise_parameter_sanitizer.permit :invite, keys: [:email, :team_members]
```

Comment:
I think this is wrong but not in the way you think its wrong. I think actually we can remove the line from reading about this online, it actually has nothing to do with user.invite!, it actually means invitation.new which isnt anything we care about. The default is email so we dont need to state it I think.